builds/get_sbt_project_directory.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+<<EOF
+Finds the path to an sbt project directory.
+
+This script is mirrored in the catalogue-pipeline and storage-service repos.
```

Comment:
Should tis be `This script is mirrored across our Scala projects.`, as it is in https://github.com/wellcomecollection/scala-libs/pull/193/files#diff-459895d3f6e6deea111c2bd33ad23854299cf6a8555794299b97222fc496dcecR5

Reply:
Good point, fixed everywhere – that's the old comment from the API repo, until I realised these scripts would end up in more places than that. 😅